src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/SessionConflict.kt

```diff
@@ -0,0 +1,7 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.model
+
+@Suppress("unused")
+enum class SessionConflict {
+  NON_ASSOCIATION,
```

Comment:
is there a better name for this?

Reply:
its an umbrella term currently used, i agree but i think changing it could cause more confusion.